components/common/listingCart/multipleItemsCart/ListingCartItem.vue

```diff
@@ -22,7 +22,8 @@
 
           <div class="is-flex is-align-items-end">
             <ListingCartPriceInput
-              v-model="listingCartStore.getItem(nft.id).listPrice" />
+              :model-value="listingCartStore.getItem(nft.id)?.listPrice"
```

Comment:
I would make a computed value and use `v-model`

Reply:
I looked into it but v-model was not happy about taking a computed function with a parameter. Do you maybe have a lead to do something like that?

Reply:
Best scenario I would do something like:
```ts
const listingCartItem = computed({
  get: () => listingCartStore.getItem(props.nft.id)?.listPrice,
  set: (value) => listingCartStore.setItemPrice({ id: props.nft.id, value }),
})
```

Reply:
🤯 Now I feel silly. 🤣 Not sure why I thought it needed to take a param. Will fix.